scripts/alert_emails_report/generate-subscriber-emails-file.ts

```diff
@@ -0,0 +1,21 @@
+import _ from 'lodash';
+import { fetchAllAlertSubscriptions } from '../alert_emails/firestore';
+import { getFirestore } from '../common/firebase';
+
+async function main() {
+  const db = getFirestore();
+  const subscriptions = await fetchAllAlertSubscriptions(db);
+  const fs = require('fs');
+  let output = '';
+  subscriptions.map(subscription => {
+    output = output.concat(subscription.email, ',');
+  });
+  fs.writeFileSync('subscriber-emails.txt', output);
```

Comment:
You may want to check with Mikayla / Anya (or the mailchimp docs) about what formats mailchimp will import.  We might need a single email per line or something (which would basically be a valid CSV file as long as you added a single "email" header row to the top).

Reply:
Just checked with Mikayla, this text file format should be fine 👍🏼